saturn-datepicker/src/datepicker/calendar-body.ts

```diff
@@ -218,7 +225,7 @@ export class SatCalendarBody implements OnChanges {
 
   /** Whenever to mark cell as end of the range. */
   _isEnd(date: number): boolean {
-    if (this.rangeMode && this.beginSelected && this._cellOver) {
+    if (['range', 'since', 'until'].includes(this.selectionMode) && this.beginSelected && this._cellOver) {
```

Comment:
add array to the constant

Reply:
This is right and needed, as we have multiple types of ranges now.